### PR TITLE
Adding ta-lib

### PR DESCRIPTION
**Pull request by richafrank**
Hi @conda-forge/help-python-c 

My goal here is to add a package for the [ta-lib python wrapper](https://github.com/mrjbq7/ta-lib) for the [TA-Lib c/c++ library](https://ta-lib.org/hdr_dw.html), essentially porting the recipe we've had [here](https://github.com/quantopian/zipline/tree/5ab89ff25d3ea99e8700e2343496d15eec67d0bd/conda/ta-lib) to conda-forge. 

My thought was to split the c lib into its own package now ("libta-lib"), whereas our original recipe in the zipline repo inlines that installation in the wrapper package. Any recommendations on this? I believe this necessitates `run_exports`, but feedback would be most appreciated.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/libta-lib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://downloads.sourceforge.net/project/ta-lib/ta-lib/0.4.0/ta-lib-0.4.0-src.tar.gz  # [unix]
+    sha256: 9ff41efcb1c011a4b4b6dfc91610b06e39b1d7973ed5d4dee55029a0ac4dc651                    # [unix]
+    folder: unix                                                                                # [unix]
+  - url: https://downloads.sourceforge.net/project/ta-lib/ta-lib/0.4.0/ta-lib-0.4.0-msvc.zip
+    sha256: 2dae7a1391222f0a7e601ecda654f93cebedfdb37afeb30df2721a3b0a717875
+    folder: win
```
Comment (richafrank) [inline recipes/libta-lib/meta.yaml:15]: The LICENSE file is only found in the windows zip, so I'm downloading it in all cases to find that.

File: recipes/libta-lib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  home: https://ta-lib.org/index.html
+  summary: "TA-Lib : Technical Analysis Library"
+  license: BSD-2-Clause
+  # License is included in windows source download, but not unix:
+  license_file: win/LICENSE.TXT
+
+extra:
+  recipe-maintainers:
+    - gerrymanoim
+    - llllllllll
```
Comment (scopatz) [inline recipes/libta-lib/meta.yaml:53]: @gerrymanoim @llllllllll - are you OK with being maintainers?

Comment (gerrymanoim) [reply recipes/libta-lib/meta.yaml:53]: Yep - sorry missed this. All good.

Comment (llllllllll) [reply recipes/libta-lib/meta.yaml:53]: I am.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**